pkgs/tools/misc/zthrottle/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, zsh }:
+
+stdenv.mkDerivation rec {
+  pname = "zthrottle";
+  version = "git-2017-7-24";
+
+  src = fetchFromGitHub {
+    owner = "anko";
+    repo = "zthrottle";
```

```suggestion
    repo = pname;
```
opinionated change :p